package.json

```diff
@@ -10,5 +10,8 @@
     "postinstall": "cd client && yarn install",
     "rails-server": "echo 'visit http://localhost:3000/hello_world' && foreman start -f Procfile.dev",
     "test": "rspec"
+  },
+  "dependencies": {
+    "babel-loader": "^6.4.1"
```

Comment:
I mentioned this in the [google doc](https://shakacode.slack.com/files/justin/F4T6MPS8N/react_on_rails_with_webpacker) webpacker seems to look for the babel-loader in the root, not in /client, even though I set it in the config. I've added it here temporarily to get the example working.